chapter14/kds/ex1.pl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+chomp( my @numbers = <STDIN> );
+
+print "------\n";
+printf "%10g\n" x @numbers, ( sort {$a <=> $b} @numbers );
```

Comment:
I remember `x` is used as replicating, so I guess here it will print every element of array, Neat!